stock_landed_costs_auto_create/models/purchase_order.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    landed_costs_ids = fields.One2many(
+        comodel_name="stock.landed.cost",
+        inverse_name="purchase_id",
+        string="Landed Costs",
+    )
+
+    def _prepare_slc_values(self):
+        return {
+            "purchase_id": self.id,
+        }
+
+    def _create_picking(self):
+        res = super()._create_picking()
+        for order in self:
+            self.env["stock.landed.cost"].with_context(
+                company_id=order.company_id.id
```

Comment:
Why in context and not with_company() ?

Reply:
Sorry but `.with_company()` is available in v14.

Reply:
Shouldn't it be `force_company` context anyways?

Reply:
@victoralmau you left this unattended.

Reply:
Ok, sorry, I forgot about this change.